Update Testerina guides (#3314)
ballerina-platform/ballerina-dev-website: learn/guides/testing-ballerina-code/writing-tests.md

### '@test:Config {}'

The function specified after the annotation is a test function. This annotation supports fields to configure the test case with the following.

*   Enable/disable the test.
    ***enable: {true&#124;false}*** - Default: true
    

> *   Setup and tear down prerequisites required for a test case.

Reviewer: ```suggestion
*   Set up and tear down the prerequisites required for a test case.
```

### '@test:Config {}'

The function specified after the annotation is a test function. This annotation supports fields to configure the test case with the following.

*   Enable/disable the test.

>     ***enable: {true&#124;false}*** - Default: true

Reviewer: ```suggestion
    ***`enable: {true&#124;false}`*** - The default value is `true`.
```
Please check backticks in all occurrences.

    
*   Set up and tear down prerequisites required for a test case.

>     ***before: &lt;function name&gt;*** - The function to be run just before the test is run. Default: none

Reviewer: ```suggestion
    ***before: &lt;function name&gt;*** - The function to be run just before the test is run. The default value is `none`.
```

>     ***after: &lt;function name&gt;*** - The function to be run just after the test is run. Default: none

Reviewer: ```suggestion
    ***after: &lt;function name&gt;*** - The function to be run just after the test is run. The default value is `none`.
```

> *   Define the order of test execution.

Reviewer: ```suggestion
*   Defines the order of the test execution.
```

    ***dependsOn: [&lt;function names>, …]*** - List of functions on which the test function depends. The order in which the comma-separated list appears has no prominence. In case there needs to be an order, the `dependsOn` parameter can be used to create an ordered sequence of functions with one function depending on the other.

> *   Define data sets for data-driven tests.

Reviewer: ```suggestion
*   Define data sets for data-driven tests.
```
Are these titles? Better if we can make them sub titles.

>     ***dataProvider: &lt;function name>*** - Specifies the function that will be used to provide the data sets for the test.

Reviewer: ```suggestion
    ***dataProvider: &lt;function name>*** - Specifies the function, which will be used to provide the data sets for the test.
```

    </tr>
</table>

> ### Troubleshooting assertion failures when using 'assertEquals'

Reviewer: ```suggestion
### Troubleshooting Assertion Failures when Using 'assertEquals'
```

> #### Values with different types

Reviewer: ```suggestion
#### Values with Different Types
```
Please check the title case in all the titles.
